.gitignore

```diff
@@ -0,0 +1,2 @@
+dist/bundle.js
```

Comment:
I think we can just ignore dist folder as a whole, since we might add more files besides bundle.js
(e.g. static files, images)